docs/presets.rst

```diff
@@ -153,6 +153,15 @@ it into an MP4 container:
 
 * ``best_video_quality``
 
+
+Limit maximum quality to 1080P (No 4K)
```

Comment:
Let's rename this title to `Max 1080p`

Reply:
That is for the existing preset that was there.  not my addition.

Best Video Quality
^^^^^^^^^^^^^^^^^^

Add the following preset to download the best available video and audio quality, and remux
it into an MP4 container:

* ``best_video_quality``


Limit maximum quality to 1080P (No 4K)
^^^^^^^^^^^^^^^^^^

Add the following preset to download the best available (but not greater then 1080P video) and audio quality
The output container MP4:

* ``max_1080p``

Reply:
I'm saying let's make the title you added simpler, from 'Limit maximum quality to 1080P (No 4K' to just 'Max 1080p'